osi_trafficcommand.proto

```diff
@@ -461,8 +480,181 @@ message SpeedAction
 
 }
 
+// \brief Longitudinal Distance Action
+//
+// The action tells the traffic participant to reach a longitudinal distance to 
+// another traffic participant. The longitudinal distance is defined as the distance
+// along the lane. 
+//
+// \note This action is aligned with LongitudinalDistanceAction of OpenSCENARIO 1.0
+// defining the reference traffic participanz and the distance.
+//
+
+message LongitudinalDistanceAction 
+{
+    // Action Header of the message
+    //
+	optional ActionHeader action_header = 1;
+	
+    // Unique Id of the refence traffic participant to which the distance is defined.
+    //
+	optional Identifier target_traffic_participant_id = 2;
+	
+	// The distance to reach along the lane
+	//
+	// Unit: m
+	//
+	optional double distance = 3;
+	
+	// Determine the measurement of distance between TrafficParticipant
+	// True: Lateral distance is measured using the distance between closest bounding box points.
+    // False: Reference point distance is used
```

Comment:
I don't understand what is meant with "reference point distance". I see three options (there might be more): 
1) explain it better and allow an option to set the reference point(s) 
2) say the other option is center of bounding box to center of bounding box
3) don't allow options and force the user to define the distance with freespace

Reply:
Resolved in [a71b8cf](https://github.com/OpenSimulationInterface/open-simulation-interface/pull/453/commits/a71b8cf81ad6a7f0b8124ab8cd4e9d75e0766cad)